src/utils/utils/format.js

```diff
@@ -0,0 +1,3 @@
+export function financial(x) {
```

Comment:
Good way to format your currency data, as a sugestion you could check the NumberFormat function of JS
```
export const currencyFormatter = new Intl.NumberFormat("en-US", {
  style: "currency",
  currency: "USD",
  minimumFractionDigits: 2,
});
```